packages/integration-sdk-runtime/src/api/index.ts

```diff
@@ -18,6 +19,7 @@ interface CreateApiClientInput {
   account: string;
   accessToken?: string;
   retryOptions?: RetryOptions;
+  lambda?: typeof Lambda;
```

Comment:
Confusingly, the alpha client options include both `lambda` (a callback function) and `Lambda` (the class). I recommend using `Lambda` as the argument in case we want to extend and use `lambda` at some point in the future.